docs/DeveloperGuide.md

```diff
@@ -1272,3 +1272,27 @@ testers are expected to do more *exploratory* testing.
    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+### Resizing the Help Window
+
+1. Resizing the Help Window
+
+   1. Prerequisite: Help Window is open
+
+   1. Test case: Resize Help Window to be smaller <br>
+   Expected: The Help Window's dimesions will not go below `1000x600`.
+
+   1. Test case: Resize Help Window to be larger <br>
+   Expected: The Help Window's dimesions will not go above `1250x700`.
+
+### Resizing the Onboarding Guide prompt
+
+1. Resizing the Onboarding Guide prompt
+
+   1. Prerequisite: Onboarding Guide prompt is open
+
+   1. Test case: Resize Onboarding Guide prompt to be smaller <br>
+   Expected: The prompt's dimesions will not go below `150x500`.
```

Comment:
Should be `500x150` (width x height). 

Also, I think the expected result can be written along the lines of:
```suggestion
   Expected: The prompt's dimensions will not go have width smaller than 500 or height smaller than 150.
```
If anyone have a better proposal for the phrasing feel free to suggest!

Reply:
Alright changed all to width x height. Would like to change to explicit bt dk if there is a unit width and height. like 500px or smth